test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
So the tests right now are (should) be in alphabetical order. I think that "h" comes alphabetically before "help", no?